char_index.py

```diff
@@ -0,0 +1,29 @@
+char_position = 0
```

Comment:
this variable is never used and could be confused with the local variable created within the method.
```suggestion
```

Reply:
Don't we still want to declare for use here later? 
 return return_message.format(char_position=ord(character_argument) - 96)

Reply:
nope declaring vars before we use them is for shitty languages from the 80's